ios/FluentUI/IndeterminateProgressBar/IndeterminateProgressBar.swift

```diff
@@ -45,6 +45,11 @@ public struct IndeterminateProgressBar: View, TokenizedControlView {
                 :
                 "Accessibility.ActivityIndicator.Stopped.label".localized
         }()
+#if DEBUG
+        let accessibilityIdentifier: String = {
+            return "Indeterminate Progress Bar that is \(state.isAnimating ? "in progress" : "progress halted")"
+        }()
```

Comment:
optional I think you could do `let accessibilityIdentifier: String = "Indeterminate..."